screen.css

```diff
@@ -0,0 +1,41 @@
+.table{
+    border:4px solid blue;
+    width:100%;
+    height:100%;
+    font-size:50px;
+  }
+  .first_row td{
+    border:4px solid black;
+    height:12%;
+    width:49%;
```

Comment:
some fonts are bigger some are smaller

Reply:
all the font is in the 50px

should I change the font size